File: applications/summersim18/src/python/div.py
if __name__ == '__main__':
    a = 100
    b = 1e308
    c = 11
    d = (a / b) / c
    print("(100 / 1e308) / 11 = %01.20e" % d)

    d = a / (b * c)
    print("100 / (1e308 * 11) = %01.20e" % d)

    d = a / b / c
    print("100 / 1e308 / 11 = %01.20e" % d)

File: applications/summersim18/src/cpp/div.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <iostream>
#include <iomanip>

int main()
{
    double a,b,c,d;
    a = 100;
    b = 1e308;
    c = 11;

    d = (a / b) / c;
    std::cout << "(100 / 1e308) / 11 = " << std::setprecision(20) << d << std::endl;

    d = a / (b * c);
    std::cout << "100 / (1e308 * 11) = " << std::setprecision(20) << d << std::endl;

    d = a / b / c;
    std::cout << "100 / 1e308 / 11 = " << std::setprecision(20) << d << std::endl;

    return 0;
}
